CMakeLists.txt

```diff
@@ -129,6 +129,12 @@ set(${PROJECT_NAME}_TASKS_HEADERS
     include/tsid/tasks/task-capture-point-inequality.hpp
     include/tsid/tasks/task-angular-momentum-equality.hpp)
 
+
+set(${PROJECT_NAME}_MEASUREDFORCES_HEADERS
+    include/tsid/measuredForces/measured-force-base.hpp
+    include/tsid/measuredForces/measured-force-3Dforce.hpp
+    include/tsid/measuredForces/measured-force-6Dwrench.hpp)
```

Comment:
```suggestion
    include/tsid/measured-forces/measured-force-base.hpp
    include/tsid/measured-forces/measured-force-3Dforce.hpp
    include/tsid/measured-forces/measured-force-6Dwrench.hpp)
```
Using this convention seems to be aligned sith the rest of the project.

Reply:
Maybe, putting that in contacts make more sens. I think we need the opinion of @andreadelprete here.

Reply:
Yes, I think that it'd make more sense to put it in the contact namespace

Reply:
Okay, I'm going to change it into "contacts" namespace

Reply:
I did change it to Contact namespace, please tell me if it's okay for you like that

Reply:
Yes, that's ok for me